src/KM_Controls.pas

```diff
@@ -2491,8 +2612,8 @@ function TKMEdit.KeyEventHandled(Key: Word; Shift: TShiftState): Boolean;
     VK_END: Result := (fText <> ''); //These keys have no effect when text is blank
   end;
 
-  //We want these keys to be ignored by chat, so game shortcuts still work
-  if Key in [VK_F1..VK_F12, VK_ESCAPE] then Result := False;
+  //We want these keys to be ignored by TKMEdit
+  if Key in [VK_F1..VK_F12, VK_ESCAPE, VK_RETURN, VK_TAB] then Result := False;

```

Comment:
After a quick read :
Can you still send a chat message by pressing the return key? I'm quite sure that a lot of players enjoy that feature.
Should be tested to be sure. :-)